labs/lab-1-html-and-css/stylesheet.css

```diff
@@ -0,0 +1,200 @@
+body {
+  max-width: 900px;
+  margin: auto;
+  background-color: #f9fafc
+}
+
+.main{
+  position: relative;
+  /* background-color: green; */
```

Comment:
In the future, give your files a once-over and ditch these debugging rules before you push for the last time.